index.js

```diff
@@ -182,3 +182,20 @@ projectButton.forEach((el) => el.addEventListener('click', () => {
     closeTag.onclick = closeIt;
```

Comment:
- [ ] [OPTIONAL] You have done a good job with the form validation :100: but I think it could be a good idea that once the user corrects the errors the message disappears, so the user knows that the error it's already solved :pushpin:

Reply:
Yeah. I get that. Thank you very much. Am going to deal with it.